workers/grouper/src/index.ts

```diff
@@ -187,6 +196,11 @@ export default class GrouperWorker extends Worker {
         },
       });
     }
+
+    /**
+     * Unlock event lock, because we've finished processing it
+     */
+    await this.redis.unlockEvent(uniqueEventHash, task.event.user?.id);
```

Comment:
Please, explain this statement. It's unexpected to see some "unlock" in this method because it does not have "lock"

Reply:
removed